src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM.java

```diff
@@ -140,10 +137,11 @@ public BitbucketSCM(
 
     /**
      * Regenerate SCM by looking up new repo URLs etc.
+     *
      * @param oldScm old scm to copy values from
      */
     public BitbucketSCM(BitbucketSCM oldScm) {
-        this(oldScm.getId(), oldScm.getBranches(), oldScm.getCredentialsId(), oldScm.getExtensions(),
+        this(oldScm.getId(), oldScm.getBranches(), oldScm.getCredentialsId(), emptyList(),
```

Comment:
Is this change necessary? What's the harm in still passing the extensions to the new instance (even if we don't have any extensions at the moment)?